server/app.py

```diff
@@ -1,12 +1,11 @@
 import logging
 import os
+
 from flask import Flask, send_from_directory, redirect, request
 from flask_jwt_extended import JWTManager, jwt_required
 from flask_sqlalchemy import SQLAlchemy
 from flask_wtf.csrf import CSRFProtect
 from werkzeug.exceptions import BadRequestKeyError
-
-from execution.api import location, patient, actions
 from execution.utils import util
 from execution.entities.execution import Execution
```

Comment:
By convention, imports are grouped by standard libraries, external libraries and internal imports, no need to change that here.

Reply:
this import moved to line 24 as asynch import